subprojects/configuration-cache/src/main/kotlin/org/gradle/configurationcache/ConfigurationCacheReport.kt

```diff
@@ -67,6 +67,7 @@ class ConfigurationCacheReport {
         }
     }
 
+    @Suppress("DEPRECATION") // TODO Fix once we build with Kotlin 1.4
```

Comment:
Fixing those will require building Gradle's source code with `apiVersion` and `languageVersion` to 1.4.
Leaving that for a subsequent PR.